src/Congo.WebApi/Controllers/ProductsController.cs

```diff
@@ -28,12 +29,29 @@ public async Task<IEnumerable<Product>> Get()
             return await _mediator.Send(new GetProductListQuery());
         }
 
+
+        [HttpPost]
+        [Route("~/api/{seller}/products")]
+        public ActionResult<Product> Create(ProductInputModel product)
+        {
+            if (!ModelState.IsValid)
```

Comment:
The validation happens automatically so you do not need to manually write it out in a Web API unlike f.e MVC